package.json

```diff
@@ -67,5 +67,6 @@
       "keccak": true,
       "fsevents": true
     }
-  }
+  },
+  "packageManager": "yarn@1.22.19"
```

Comment:
This is to ensure that correct Yarn version is used for people with Yarn >=2 installed. Can be changed when we migrate this to Yarn 3/4.